packages/core-types/cardstack/field-types/fixed-decimal.js

```diff
@@ -0,0 +1,8 @@
+module.exports = {
+  valid(value) {
+    return typeof value==='number';
+  },
+  buildQueryExpression(sourceExpression, name){
```

Comment:
I honestly don't know what this function does.

Reply:
This is needed because of how types work in postgres. If somebody is trying to write a `filter` or `sort` that depends on a fixed-decimal field, we need to know how to exact that value out of a JSON document.

`sourceExpression` is some postgres expression that results in the JSON for a model and `name` is the name of a fixed-decimal field that we want to extract from that model.

The [`->>` operator is documented here](https://www.postgresql.org/docs/10/functions-json.html#FUNCTIONS-JSON-OP-TABLE). It extracts the value from the json, but as text. So it also needs to be cast to a number, which you're doing with `::real`.

Postgres does have a native `decimal` type that might also work here instead of `real`.